candle-metal-kernels/src/unary.metal

```diff
@@ -19,7 +19,9 @@ METAL_FUNC uint get_strided_index(
 }
 
 template <typename T> METAL_FUNC T sqr(T in){ return in * in; }
+template <typename T> METAL_FUNC T urecip(T in){ return T(1.0 / in); }
```

Comment:
This is the inverse op, why call it urecip ?

`inv` seems like a better name. (`u` is not necessary)

Reply:
Candle names this op recip. I could change all instances to inv as i also agree it's a better name but I the Rust library itself names it recip so that would be more consistent. Either name works for me.